run_dir/design/project_samples.html

```diff
@@ -206,6 +206,10 @@
                     <td><span class="badge" id="charon-status-tot"></span></td>
                 </tr>
                 <tr>
+                    <td>Aborted samples</td>
+                    <td><span class="badge" id="charon-status-ab"></span></td>
```

Comment:
Fixed in https://github.com/SciLifeLab/genomics-status/pull/322/commits/bf861fe308d7caf878eb58b7488f8cc63858bb31